docs/framework/react/guide/file-based-routing.md

```diff
@@ -77,6 +77,27 @@ export default defineConfig({
 
 > ⚠️ **Note:** The `experimental.enableCodeSplitting` option is not yet supported in Rspack/Rsbuild.
 
+### Configuration with Webpack
+
+To use file-based routing with **Webpack**, you'll need to install the `@tanstack/router-plugin` package.
+
+```sh
+npm install -D @tanstack/router-plugin
+```
+
+Once installed, you'll need to add the plugin to your configuration.
+
+```tsx
+// webpack.config.ts
+import { TanStackRouterWebpack } from '@tanstack/router-plugin/webpack'
+
+export default {
+  plugins: [TanStackRouterWebpack()],
+}
+```
+
+> ⚠️ **Note:** The `experimental.enableCodeSplitting` option is not yet supported in Webpack.
```

Comment:
Also this is not true anymore, same goes for rspack

Reply:
Yup!